.github/ISSUE_TEMPLATE/new-invited-expert.md

```diff
@@ -11,14 +11,15 @@ assignees: lucacasonato, andreubotella
 
 Hi! Welcome to WinterTC! Please fill out this form to start the process to become an invited expert.
 
+In addition to this form, you also need to fill Ecma's Invited Expert form (https://ecma-international.org/invited-expert-form/), selecting "TC55" as the technical committee to participate in. You need to do this even if you already are an invited expert for any other TCs.
+
 -->
 
 - Name: 
 - Pronouns: 
 - Affiliation: 
 - Reason for becoming an invited expert: 
 - Matrix username: 
-- Email (optional): 
-  <!-- Please remember that this issue will be public! If you don't want to share your email address publicly, don't write anything here. The WinterTC chairs will ask you to join on Matrix, and ask you for your email over private messsage. -->
 
-- [ ] I claim that this information is correct, and that I do not belong to an Ecma member organization.
\ No newline at end of file
+- [ ] I claim that this information is correct, and that I do not belong to an Ecma member organization.
```

Comment:
Small note: When there are empty lines between two lists, github renders all items with extra space. Adding a title before this line helps fixing this issue.

Reply:
Thanks for pointing it out! Opened #90 to fix this